### PR TITLE
Dependency check for `iconv` library modified

### PR DESCRIPTION
**Pull request by sambitdash**
Added `libc-bin` to the alias list as that is how `libc` reported on Ubuntu 17.04.

### CODE REVIEW

**Comment by sambitdash** [comment]
This fixes issue #18.

File: README.md [hunk 1]
```diff
@@ -149,3 +149,7 @@ julia> readstring(s) # Decoding happens automatically here
 Do not forget to call `close` on `StringEncoder` and `StringDecoder` objects to finish the encoding process. For `StringEncoder`, this function calls `flush`, which writes any characters still in the buffer, and possibly some control sequences (for stateful encodings). For both `StringEncoder` and `StringDecoder`, `close` checks that there are no incomplete sequences left in the input stream, and raise an `IncompleteSequenceError` if that's the case. It will also free iconv resources immediately, instead of waiting for garbage collection.
 
 Conversion currently raises an error if an invalid byte sequence is encountered in the input, or if some characters cannot be represented in the target enconding. It is not yet possible to ignore such characters or to replace them with a placeholder.
+
+## Notes on Installation on Linux OS
+
+Most Linux distributions provide `iconv` functionalities as part of the base operating system library `libc`. In normal circumstances, no additional installation of `libiconv` may be required by downloading it from the GNU repository and building from the sources. If you observe such a behavior on your operating system, file an issue with OS details.
```
Comment (nalimilan) [inline README.md]: "libc" -> "glibc". "may" -> "should". Also no need to say "by downloading it from the GNU repository and building from the sources" (the remarks also apply to installing from a package).

File: deps/build.jl [hunk 2]
```diff
@@ -31,7 +31,7 @@ function validate_iconv(n, h)
     return ret == -1 % Csize_t && Libc.errno() == Libc.EILSEQ
 end
 
-libiconv = library_dependency("libiconv", aliases = ["libc", "iconv"],
+libiconv = library_dependency("libiconv", aliases = ["libc", "iconv", "libc-bin"],
```
Comment (nalimilan) [inline deps/build.jl]: Better put `libc-bin` before `iconv`.

**Comment by nalimilan** [comment]
Thanks!

**Pull request merged**